src/components/Popups/__tests__/PopupDrawingToolbar-test.tsx

```diff
@@ -109,7 +128,7 @@ describe('PopupDrawingToolbar', () => {
             ${'onReply'}  | ${'comment'}
         `('should call $callback when $button is clicked', ({ callback, buttonId }) => {
             const mockFn = jest.fn();
-            const wrapper = getWrapper({ [callback]: mockFn });
+            const wrapper = getWrapper({ [callback]: mockFn, canRedo: true, canUndo: true });
```

Comment:
Added this change because tests broke after we introduced mount (the buttons are disabled if we don't explicitly set `canRedo` and `canUndo` to true.